mozilla/experimenter
# Use ext for eslint

I'd recommend using the `ext` syntax, which also removes the need for `--no-error-on-unmatched-pattern`, i.e.`--ext=.js,.jsx,.ts,.tsx ` v.s. `./*.ts ./*.tsx ./**/*.ts ./**/*.tsx ./**/*.js` 



There are some inconsistencies between ts and js linting at the moment (@pdehaan mentioned on in other patches) that can be resolved with changes to `.eslintrc.js` but i can file those as a follow-up

Review steps:
1. Pull `k88hudson:eslint-rapid-errors`
2. Try linting, add your own errors if you want. 
3. Ensure you see the lint task fail for warnings, errors
